src/widget/OktaSignIn.js

```diff
@@ -127,7 +130,22 @@ var OktaSignIn = (function () {
       Router = V1Router;
     }
 
-    _.extend(this, Router.prototype.Events);
+    _.extend(this, Router.prototype.Events, {
+      on: function (...onArgs) {
+        // custom events listener on widget instance to trap third-party callback errors
+        const [event, callback] = onArgs;
+        if (EVENTS_LIST.includes(event)) {
+          onArgs[1] = function (...callbackArgs) {
+            try {
+              callback.apply(this, callbackArgs);
+            } catch (err) {
+              DEBUG && Logger.error(`[okta-signin-widget] "${event}" event handler error:`, err);
```

Comment:
@pradeepdewda-okta Comment addressed

Reply:
Isn't the intent to show error at product build so that developer can discover the error during development?